src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -143,13 +140,23 @@ protected override void EndProcessing()
             }
 
             base.EndProcessing();
+        }
 
-            if (Transcript)
+        /// <summary>
+        /// Revert transcription state on Dispose
+        /// </summary>
+        protected override void InternalDispose()
```

Comment:
Are not you supposed to call `base.InternalDispose()` at some point of overriding method?